gatsby-node.js

```diff
@@ -295,6 +307,12 @@ exports.createSchemaCustomization = ({ actions }) => {
       TotalContributions: String
     }
 
+    type Referendum implements Node {
+      Title: String!
+      Description: String
+      Total_Contributions: String
```

Comment:
Is this what the API returns? Can we ask to remove the underscore for consistency?

Reply:
So the API returns a `"Total Contributions"` key, I wasn't sure how to put that in since it threw an error when I used quotes, and I think it threw an error when I put it in two words too? I couldn't find how to define a string like that but I saw the _ being used elsewhere and it worked on the Referendums, so I left it in. Maybe we should get @geleazar1000111 to change the field to `TotalContributions`?

Reply:
Yeah sounds like we should get rid of the space on the backend if we can